src/main/scala/sbtecr/EcrPlugin.scala

```diff
@@ -7,6 +7,8 @@ import sbt._
 import scala.language.postfixOps
 
 object EcrPlugin extends AutoPlugin {
+  
+  lazy val version:SettingKey[String] = settingKey[String]("The ECR docker image version.")
```

Comment:
Technically speaking, we don't need `sbt-native-packager` or the `DockerPlugin` to work. I wanted to avoid this dependency (at least initially), so that the image build is independent from the publishing step (i.e. as long as "some" Docker image exist in the local repo,  the publishing part should work and cover all required steps; we're not concerned about getting the image there though). 

I'm not very fluent with `sbt` internals myself, but from what I see, it's exactly how it works in the [DockerPlugin](https://github.com/sbt/sbt-native-packager/blob/master/src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala#L18) - i.e the `version` setting is imported from the `Keys` object. 

Please feel free to give it a shot - I'll gladly test it myself once you update the PR.